tools/graph_analyser/analyser.py

```diff
@@ -42,23 +74,63 @@ def print_search(graph, start, end):
 
         # Remove duplicate edges
         sgraph = strictify(graph)
-        
+
+        if dfs(sgraph, 'remote', 'kernel'):
+            grid[vindex, dindex] = 17
```

Comment:
What is the significance of this reverse sequence of odd numbers?

Reply:
Intended so they are unambiguously between the (even number) "limits" for each of the categories, but probably not really necessary.

Reply:
Yes I worked that out as I read on, it is a good strategy, but a comment at this point would help clarify to the future reader.